draft-ietf-taps-arch.md

```diff
@@ -182,6 +182,7 @@ Providing a message-based abstraction provides many benefits, such as:
 * the ability to associate deadlines with messages, for applications that care about timing;
 *  the ability to provide control of reliability, choosing which messages to retransmit in the event of packet loss, and how best to make use of the data that arrived;
 * the ability to manage dependencies between messages, when the transport system could decide to not deliver a message, either following packet loss or because it has missed a deadline. In particular, this can avoid (re-)sending data that relies on a previous transmission that was never received.
+* the ability to automatically assign messages and connections to underlaying transport connections to exploit multi-streaming and pooled connections.
```

Comment:
I don't love using the word `exploit` here; it's a bit of a negative connotation. How about using `utilize`?

Reply:
yep, sounds better. applied